File: TopInterview/MergeSortedArray.py
indexSec = 0
        for index in range(m+n):
            if(indexSec == n):
                break
            if nums1[index] == nums2[indexSec]:
                nums1.insert(index, nums2[indexSec])
                nums1.pop(len(nums1)-1)
                index = index - 1
                indexSec = indexSec + 1
            elif nums1[index] > nums2[indexSec]:
                nums1.insert(index, nums2[indexSec])
                index = index - 1
                indexSec = indexSec + 1
                nums1.pop(len(nums1)-1)
        for index in range(indexSec,n):
            nums1[m+index] = nums2[index]

File: TopInterview/MergeSortedArray.cs
        var nums1Index = m-1;
        var nums2Index = n-1;
        var index = m+n-1;

        while (nums2Index >= 0)
        {
            if (nums1Index >= 0 && nums1[nums1Index] > nums2[nums2Index])
            {
                nums1[index--] = nums1[nums1Index--];
            }
            else
            {
                nums1[index--] = nums2[nums2Index--];
            }
        }